src/lava/utils/float2fixed.py

```diff
@@ -1,26 +1,719 @@
 # Copyright (C) 2021-22 Intel Corporation
 # SPDX-License-Identifier: BSD-3-Clause
 # See: https://spdx.org/licenses/
-
 """
-This module will contain a tool to perform automated float- to fixed-point
-conversion of of Lava Processes.
-
-Given a set of processes and data, the tool will perform the following steps:
-
-1. Find fixed-point ProcessModels of every floating-point ProcessModel if
-available and throw an error otherwise.
-2. Run the floating-point model and collect activation statistics.
-3. Given activation statistic and type/precision/dynamic range specifications
-of fixed-point models, perform input parameter conversion for fixed-point model.
-4. Replace floating-point ProcessModels by corresponding fixed-point
-ProcessModels
-5. Initialize fixed-point ProcessModels
-
-ProcessModel extraction might be a shared function with compiler.
-
-This will likely lead to some performance degradation of the application so
-perhaps some iterative optimization may have to be included within this
-parameter conversion process. Reuse other libraries for this purpose where
-possible.
- """
+This module containx a tool to perform automated float- to fixed-point
+conversion of Lava Processes.
+"""
```

Comment:
A general remark:
As a user, I may often not know what the ideal quantiles are for my variables. That strongly depends on the distribution of the recorded data. Maybe it is worth to add a method that - after recording - plots the recorded distribution and allows the user to visually choose a quantile.
I guess such a util is out of scope for your internship. If so, maybe add a github issue instead.

Reply:
Addressed in e29f938e480575f0dd5e535280d0ce4271f3a1c8